docs/pr_checklist.md

```diff
@@ -127,3 +130,9 @@ There are instructions on how to keep your fork updated here:
 
 Thanks for contributing!
 ```
+
+## Review Process
+
+In general, we want to see two (or more) approvals, that are meaningful (eg that have gone over code) before a PR will be merged.  This is not limited to collaborators, but any community member willing to put in the time.  The only difference is that your checkmark won't be green, and that's fine! 
+
+Additionally, PR Reviews are something that is done in our free time.  We are not paid nor compensated for the time we spend reviewing, but it is a labor of love.   However, this means that it can take time for us to get to your Pull Request.  Things like family, or life can get in the way of us getting to PRs.  And burnout is a serious concern.  We average 200 PRs opened, and 200 PRs merged every month.  That's not a small amount, so please have patiences with us.  And if yuo have free time, anyone can review PRs. 
```

Comment:
```suggestion
In general, we want to see two (or more) approvals that are meaningful (e.g. that have inspected code) before a PR will be considered for merge. These reviews are not limited to collaborators -- any community member willing to put in the time is welcomed (and encouraged). The only difference is that your checkmark won't be green, and that's fine! 

Additionally, PR reviews are something that is done in our free time. We are not paid nor compensated for the time we spend reviewing, as it is a labor of love. As such, this means that it can take time for us to get to your Pull Request.  Things like family, or life can get in the way of us getting to PRs, and burnout is a serious concern. The QMK firmware repository averages 200 PRs opened and 200 PRs merged every month, so please have patience.
```

Some cleanup and removed duplication.